examples/enable-gcp-services-simple/resources.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: redis.cnrm.cloud.google.com/v1beta1
```

Comment:
I'd suggest you to drop this file, since it doesn't affect how the function behave.

Reply:
I added this as a workaround for https://github.com/GoogleContainerTools/kpt/issues/2477. For tests, do we test with kpt built from main, if so I can drop this.

Reply:
I have removed this now we are using implicit fn config